examples/opentelemetry/Dockerfile-opentelemetry

```diff
@@ -1,8 +1,11 @@
-FROM alpine:3.12 as otelc_curl
+FROM alpine:3.13 as otelc_curl
```

Comment:
one small nit!

could you add the `@sha:` for this image, thanks

Reply:
updated